jsapp/js/components/modal.es6

```diff
@@ -118,6 +119,10 @@ class Modal extends React.Component {
         });
         break;
 
+      case MODAL_TYPES.ENCRYPT_FORM:
+        this.setModalTitle(t('Add Form Encryption'));
```

Comment:
I think a better name would be "Manage Form Encryption", as you will be able to add, edit or delete it